tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -742,3 +742,88 @@ describe("Test Application deployment based on 'clusterSelector'", { tags: '@p1'
     )
   })
 });
+
+describe('Test with disablePolling', { tags: '@p1'}, () => {
+  
+  beforeEach("Ensuring Github repo has desired amount of replicas (2)", () => {
+
+    const gh_private_pwd = Cypress.env("gh_private_pwd")
+
+    cy.exec(
+      `
+      echo -e "Cloning repo"
+      git clone https://fleetqa:${gh_private_pwd}@github.com/fleetqa/fleet-qa-examples-public.git --branch=main $PWD/fleet-qa-examples-public
+
+      echo -e "Confirming repo exist in path"
+      test -e "$PWD/fleet-qa-examples-public/disable-polling"
+      if [ $? -ne 0 ]; then
+        echo "Error: path $PWD/fleet-qa-examples-public/disable-polling does not exist"
+        exit 1
+      fi
+
+      echo -e "Ensuring replicas is 2"
+      sed -i 's/replicas: ..*/replicas: 2/g' $PWD/fleet-qa-examples-public/disable-polling/nginx.yaml
+
+      echo -e "Commiting and pushing if needed "
+      cd $PWD/fleet-qa-examples-public/disable-polling
+      git config user.email "fleet.qa.team@gmail.com"
+      git config user.name "fleetqa"
+      git add nginx.yaml
+
+      if ! git diff --quiet origin/main; 
+      then
+        echo "Changes detected from original repo. Changing replicas to 2"
+        git commit -m 'Ensuring initial number of replicas is 2'
+        git push -u origin main
+      else
+        echo -e "Nothing to push, done"
+      fi 
+    `
+    ).then((result) => {
+      cy.log(result.stdout, result.stderr);
+      cy.task("log", result.stdout);
+    });
+  });
+  
+  qase(124,
+    it("Fleet-124: Test when `disablePolling=true` Gitrepo will not sync latest changes from Github", { tags: '@fleet-124' }, () => {
+  
+      cy.fleetNamespaceToggle('fleet-local')
+      cy.clickButton('Add Repository');
+      // Pass YAML file (no previous additions of Name, urls or paths)
+      cy.clickButton('Edit as YAML');
+      cy.addYamlFile('assets/disable_polling.yaml');
+      cy.clickButton('Create');
+      cy.checkGitRepoStatus('test-disable-polling', '1 / 1', '1 / 1');
+
+      // Change replicas to 5
+      cy.exec(
+        `
+        echo -e "Ensuring replicas is 5"
+        sed -i 's/replicas: ..*/replicas: 5/g' $PWD/fleet-qa-examples-public/disable-polling/nginx.yaml
+  
+        echo -e "Commiting and pushing if needed "
+        cd $PWD/fleet-qa-examples-public/disable-polling
+        git add nginx.yaml
+  
+        if ! git diff --quiet origin/main; 
+        then
+          echo "Changes detected from original repo. Changing replicas to 5"
+          git commit -m 'Ensuring initial number of replicas is 5'
+          git push -u origin main
+        else
+          echo -e "Nothing to push, done"
+        fi 
```

Comment:
Can this be put in bash file and execute it?

Reply:
mmm. Good point. I will see if this is possible something similar to this: https://stackoverflow.com/questions/76965361/can-cypress-execute-a-shell-script-in-an-existing-terminal-window
Thanks

Reply:
Done. thanks a lot for the suggestion